source/_integrations/zha.markdown

```diff
@@ -138,7 +145,7 @@ Click on **ADD DEVICES** to start a scan for new devices.
 
 Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g.,  turn on/off lights up to 10 times, switches usually have a reset button/pin).
 
-## Troubleshooting
+# Troubleshooting
```

Comment:
The H1 (first header) is reserved for the title only. Please use an H2 at a minimum.
```suggestion
## Troubleshooting
```

Reply:
Thanks @frenck @klaasnicolaas @Adminiuga for feedback. Can I double check, do you think the list of compatible Zigbee hardware covers them all? Thanks